ci-operator/step-registry/assisted/vsphere/teardown/cluster/assisted-vsphere-teardown-cluster-commands.sh

```diff
@@ -6,6 +6,6 @@ set -o pipefail
 
 echo "************ vsphere assisted test-infra teardown cluster command ************"
 mkdir -p /home/assisted-test-infra/build/cluster/
-scp -F ${SHARED_DIR}/ssh_config -r "root@ci_machine:/home/assisted/build/terraform/*" "/home/assisted-test-infra/build/cluster/"
-for DIR in /home/assisted-test-infra/build/cluster/*; do (cd "$DIR" &&  terraform apply -destroy -input=false -auto-approve); done
-
+scp -F ${SHARED_DIR}/ssh_config -r "root@ci_machine:/home/assisted/build/terraform/*" "/home/assisted-test-infra/build/cluster/vsphere"
+cd /home/assisted-test-infra/build/cluster/vsphere/vsphere
```

Comment:
It doesn't supposed to be `cd /home/assisted-test-infra/build/cluster/vsphere` ?
```

Reply:
Nope its correct - see log at https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/33148/rehearse-33148-pull-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted/1581953583920713728/artifacts/e2e-vsphere-assisted/assisted-vsphere-teardown-cluster/build-log.txt

This probably happens due to `tar xzf` changing path